Contents/mods/Perdition/media/lua/server/Items/Perdition_distributions.lua

```diff
@@ -14,7 +14,46 @@ local function preDistributionMerge()
                 "Perdition.CleaverUpgradeManual", 1,
                 "Perdition.ShovelUpgradeManual", 1,
             }
+        },
+        counter = {
+            rolls = 4,
+            items = {
+                "Battery", 2,
+                "BatteryPack4", 1,
+                "BatteryPack8", 1,
+                "BatteryPack12", 0.7,
+                "BatteryPack16", 0.4,
+                "ACDCAdapter", 0.2,
+                "BluePen", 8,
+                "Book", 10,
+                "Eraser", 6,
+                "Magazine", 10,
+                "Notebook", 10,
+                "Paperclip", 20,
+                "PaperclipBox", 0.1,
+                "Pen", 8,
```

Comment:
You do not need to include the vanilla items as well, this script adds anything written here to the vanilla distributions